src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -110,6 +113,9 @@ def merge(self, project: Project) -> None:
 
     def _trim_rules(self, rules: RuleSet) -> RuleSet:
         sorted_rules = sorted(rules.items(), key=lambda p: p[1], reverse=True)
+        last_seen_deadline = _now() - TRANSACTION_NAME_RULE_TTL_SECS
+        sorted_rules = list(filter(lambda rule: rule[1] >= last_seen_deadline, sorted_rules))
+
         if self.MERGE_MAX_RULES < len(rules):
             with sentry_sdk.configure_scope() as scope:
                 scope.set_tag("discarded", len(rules) - self.MERGE_MAX_RULES)
```

Comment:
consider using a custom measurement instead: https://docs.sentry.io/platforms/python/performance/instrumentation/performance-metrics/#custom-metrics

Reply:
Are custom measurements indexed? Is it possible to view somehow the amount of discarded rules as a distribution? If I have to make a discover query works too.

Reply:
Addressed in https://github.com/getsentry/sentry/pull/47007.